.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -1,6 +1,6 @@
 name: 🐞 Bug report
 description: File a bug/issue
-title: "[BUG] <title>"
+title: "<component name>: <issue title>"
 labels: ["type: bug", "Needs Triage"]
```

Comment:
@agliga , @ianmcburnie , I'm assuming the `type: bug` and `type:feature` (in the other file) are good initial tags for each respectively. Is `Needs Triage` ok as well for both or do we want some other tag to indicate the new issue need to be reviewed by us?

Reply:
I think "status: needs triage" or "status: untriaged" would be good as a sensible default.

Reply:
Thanks! Updated. @agliga , @ianmcburnie , if there's nothing else can you please approve this to indicate you're good with it getting merged?